src/DatadogLoggingService.js

```diff
@@ -27,14 +27,26 @@ class DatadogLoggingService extends NewRelicLoggingService {
     this.initialize();
   }
 
-  initialize() {
+  async initialize() {
+    let datadogVersion = process.env.DATADOG_VERSION;
```

Comment:
Can we add the default value here if `process.env.DATADOG_VERSION` is undefined. This way we can remove `DATADOG_VERSION` from YAML files as it will be of no use.